src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+
+import './PostList.scss';
+
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
```

Comment:
Consider adding a fallback message when there are no posts to display, such as 'No posts available'.